modules/libcircllhist/0.3.2/patches/add_build_file.patch

```diff
@@ -0,0 +1,23 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,20 @@
++licenses(["notice"])  # Apache 2
++
++config_setting(
++    name = "windows_x86_64",
++    values = {"cpu": "x64_windows"},
++)
++
++cc_library(
++    name = "libcircllhist",
++    srcs = ["src/circllhist.c"],
++    hdrs = [
++        "src/circllhist.h",
++    ],
++    copts = select({
++        "//:windows_x86_64": ["-DWIN32"],
```

Comment:
Could you instead select on `@platforms` targets here? Otherwise this won't work correctly with platform-based toolchain resolution, which is the default in Bazel 7.

Reply:
Yes, if you want to keep the arch part of the constraints (I'm not sure why that's there though), you need to follow https://bazel.build/docs/configurable-attributes#what_should_i_do_instead.